frontend/src/components/editionOverview.tsx

```diff
@@ -0,0 +1,229 @@
+                            <a data-testid="edit-name" onClick={handleEditName}>
+                                <Image
+                                    alt={capitalize(t("edit"))}
+                                    src={"/resources/edit.svg"}
+                                    width="15"
+                                    height="15"
+                                />
+                            </a>
+                        </Col>
+                    )}
+
+                    {editName && (
+                        <Col>
+                            <input
+                                data-testid="input-name"
+                                name="name"
+                                defaultValue={data.name}
+                                onChange={onChange}
+                            />
+                            <button data-testid="save-name" onClick={handleSaveName}>
+                                <Image
+                                    alt={capitalize(t("confirm"))}
+                                    src={"/resources/checkmark.svg"}
+                                    width="15"
+                                    height="15"
+                                />
+                            </button>
+                        </Col>
+                    )}
+                </Row>
+                <Row>
+                    <Col className={styles.first_element}>{capitalize(t("year") + ":")}</Col>
+                    {/*show edition year if not editing*/}
+                    {!editYear && <Col>{year ? year : data.year}</Col>}
+                    {!editYear && (
+                        <Col>
+                            <a data-testid="edit-year" onClick={handleEditYear}>
+                                <Image
+                                    alt={capitalize(t("edit"))}
+                                    src={"/resources/edit.svg"}
+                                    width="15"
+                                    height="15"
+                                />
+                            </a>
+                        </Col>
+                    )}
+
+                    {editYear && (
+                        <Col>
+                            <input
+                                data-testid="input-year"
+                                type="number"
+                                name="year"
+                                defaultValue={data.year}
+                                onChange={onChange}
+                            />
+                            <button data-testid="save-year" onClick={handleSaveYear}>
+                                <Image
+                                    alt={capitalize(t("confirm"))}
+                                    src={"/resources/checkmark.svg"}
+                                    width="15"
+                                    height="15"
+                                />
+                            </button>
```

Comment:
Not sure but I feel like this is a component in itself?

Reply:
I felt like it wasn't generic enough to turn into a separate component, with the specific testid and onClick-function that it needs. I hope that's fine, or maybe I misunderstand your comment...